metadata-service/auth-impl/build.gradle

```diff
@@ -11,6 +11,9 @@ dependencies {
   compile project(path: ':metadata-service:auth-config')
   compile project(path: ':metadata-io')
 
+  implementation 'com.auth0:java-jwt:4.2.1'
```

Comment:
I'd really recommend that we avoid adding new dependencies. I thought we already had libraries that we use for parsing JWTs in our DataHubTokenAuthenticator. Can we reuse these?

Reply:
Yes noticed 'io.jsonwebtoken:jjwt-api:0.11.2' has similar implementation. Will us that.

Reply:
removed external dependencies.